libs/SalesforceOAuth/SalesforceOAuthUnitTests/SFOAuthFlowAndDelegate.m

```diff
@@ -101,6 +137,15 @@ - (void)beginTokenEndpointFlow:(SFOAuthTokenEndpointFlow)flowType {
     [self log:SFLogLevelDebug format:@"%@ called.", NSStringFromSelector(_cmd)];
     self.beginTokenEndpointFlowCalled = YES;
     self.tokenEndpointFlowType = flowType;
+    
+    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(5.0 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
+        if (self.refreshTokenFlowIsSuccessful) {
+            [self.coordinator handleTokenEndpointResponse:[self refreshTokenSuccessData]];
+        } else {
+            //[self log:SFLogLevelDebug format:@"%@ Error: response has no payload: %@", NSStringFromSelector(_cmd), ]
+        }
+    });
+    [self.coordinator performSelector:@selector(handleTokenEndpointResponse:) withObject:[self refreshTokenSuccessData] afterDelay:self.timeBeforeUserAgentCompletion];
```

Comment:
The `dispatch_after` block above is doing the same thing as this line.  This line can just be removed, in favor of the `dispatch_after` block doing what it's doing.